### DISCUSSION TITLE
Mailrise service fails to start

### DISCUSSION THREAD

**Discussion by dbrianhendrix** [YoRyan/mailrise#67] [Q&A]
Here is the failure I'm getting:

● mailrise.service - Mailrise SMTP notification relay
   Loaded: loaded (/etc/systemd/system/mailrise.service; disabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Thu 2023-03-30 11:32:49 CDT; 2min 5s ago
  Process: 73661 ExecStart=/usr/local/bin/mailrise /etc/mailrise.conf (code=exited, status=1/FAILURE)
 Main PID: 73661 (code=exited, status=1/FAILURE)

Mar 30 11:32:49 bde1.bluedragon.local systemd[1]: Started Mailrise SMTP notification relay.
Mar 30 11:32:49 bde1.bluedragon.local mailrise[73661]: Traceback (most recent call last):
Mar 30 11:32:49 bde1.bluedragon.local mailrise[73661]:   File "/usr/local/bin/mailrise", line 7, in <module>
Mar 30 11:32:49 bde1.bluedragon.local mailrise[73661]:     from mailrise.skeleton import run
Mar 30 11:32:49 bde1.bluedragon.local mailrise[73661]:   File "/usr/local/lib/python3.6/site-packages/mailrise/skeleton.py", line 5
Mar 30 11:32:49 bde1.bluedragon.local mailrise[73661]:     from __future__ import annotations
Mar 30 11:32:49 bde1.bluedragon.local mailrise[73661]:     ^
Mar 30 11:32:49 bde1.bluedragon.local mailrise[73661]: SyntaxError: future feature annotations is not defined
Mar 30 11:32:49 bde1.bluedragon.local systemd[1]: mailrise.service: Main process exited, code=exited, status=1/FAILURE
Mar 30 11:32:49 bde1.bluedragon.local systemd[1]: mailrise.service: Failed with result 'exit-code'.

**Comment by YoRyan** [comment]
The minimum Python version for Mailrise is 3.8 or above. Python 3.6 has been [out of support](https://devguide.python.org/versions/) since the end of 2021.
